tools/scripts/generate_docker_release_matrix.py

```diff
@@ -29,7 +29,7 @@ def generate_docker_matrix(channel: str) -> Dict[str, List[Dict[str, str]]]:
     prefix = "ghcr.io/pytorch/pytorch"
     docker_image_version = ""
     if channel == "release":
-        docker_image_version = f"{prefix}:{generate_binary_build_matrix.CURRENT_STABLE_VERSION}"
+        docker_image_version = f"{prefix.replace("ghcr.io/", "")}:{generate_binary_build_matrix.CURRENT_STABLE_VERSION}"
```

Comment:
Here we probably want to pass another flag. Something like ``generate_dockerhub_images`` since we still want to validate ghcr.io release images before uploading to dockerhub.